tastypie/bundle.py

```diff
@@ -32,4 +33,7 @@ def __init__(self,
         self.via_uri = via_uri
 
     def __repr__(self):
-        return "<Bundle for obj: '%s' and with data: '%s'>" % (self.obj, self.data)
+        repr_string = "<Bundle for obj: '%s' and with data: '%s'>"
```

Comment:
Ahh, I forgot about unicode_literals.

I just realized, probably the best solution is to use %r instead of %s.